modules/ipi-install-bmc-addressing-for-fujitsu.adoc

```diff
@@ -0,0 +1,53 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id='bmc-addressing-for-fujitsu_{context}']
+
+= BMC addressing for Fujitsu
+
+The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: <host name>
+        role: <master | worker>
+        bmc:
+          address: <address>
+          username: <user>
+          password: <password>
+----
+
+For Fujitsu hardware, Red Hat supports iRMC and IPMI.
+
+.BMC address formats for Fujitsu hardware
+[frame="topbot",options="header"]
+|====
+|Protocol|Address Format
+|iRMC| `irmc://<out-of-band-ip>`
+|IPMI| `ipmi://<out-of-band-ip>`
+|====
+
+.iRMC for Fujitsu
```

Comment:
@sferich888  Does your comment above apply here too? Should this be .iRMC for Fujitsu hardware?